libs/@guardian/atoms-rendering/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
the license at the root of the monorepo (https://github.com/guardian/csnx/blob/main/LICENSE) looks like it's the same as this one. Does the root one cover this subdirectory too I wonder? Or would it just be better to be explicit and include it twice 🤔

Reply:
Good spot, removed.